x-pack/plugins/apm/public/components/app/RumDashboard/PageLoadDistribution/ResetPercentileZoom.tsx

```diff
@@ -49,6 +50,6 @@ export function ResetPercentileZoom({
           {I18LABELS.resetZoom}
         </EuiButtonEmpty>
       </EuiHideFor>
-    </>
-  );
+    </EuiFlexItem>
```

Comment:
Nit: if we're no longer displaying the button when isDisabled is false, we no longer need to pass the disabled prop to the buttons.